apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -118,16 +119,22 @@ export KUMA_MOUNT_PATH ?= /www
 export KUMA_DEBUG ?= False
 export KUMA_DEBUG_TOOLBAR ?= False
 export KUMA_PROTOCOL ?= "https://"
+ifeq (${TARGET_ENVIRONMENT},dev)
+export KUMA_DOMAIN=$(shell kubectl get --ignore-not-found -n ${K8S_NAMESPACE} svc ${WEB_SERVICE_NAME} -o jsonpath={.status.loadBalancer.ingress[0].hostname})
+else
```

Comment:
The `regions/portland/stage.sh` setup has that defined. I'm still operating under the assumption that the env variable values in the `Makefile` are the defaults if the env variable is not defined (assuming the `dev` target environment), and that we'll use the `regions/portland/{dev|stage|prod}.sh` scripts to the the setup before using `make`.